src/contrib/persistence/Akka.Persistence.Query.Sqlite.TestKit/Akka.Persistence.Query.Sqlite.TestKit.csproj

```diff
@@ -0,0 +1,18 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Need need to add additional projects - this should all just be part of Akka.Persistence.Sqlite / .Tests